### PR TITLE
common: fix readme eip links

### PR DESCRIPTION
**Pull request by gabrocheleau**
This PR fixes a couple instances where EIP numbers linked to the wrong EIPs

### CODE REVIEW

File: packages/evm/src/types.ts [hunk 1]
```diff
@@ -210,7 +210,7 @@ export interface EVMOpts {
 
   /**
    * Allows unlimited contract sizes while debugging. By setting this to `true`, the check for
-   * contract size limit of 24KB (see [EIP-170](https://git.io/vxZkK)) is bypassed.
+   * contract size limit of 24KB (see [EIP-170](https://git.io/vxZkK)) is 170.
```
Comment (acolytec3) [inline packages/evm/src/types.ts:213]: Why did you change this?  This doesn't feel like a clear chainge to me.

Comment (gabrocheleau) [reply packages/evm/src/types.ts:213]: That one's a mistake, undoing

Comment (gabrocheleau) [reply packages/evm/src/types.ts]: Resolved

**Review by acolytec3 — APPROVED** [review]

**Review by acolytec3 — APPROVED** [review]
LGTM
